client/src/pages/singlePlace/index.js

```diff
@@ -0,0 +1,175 @@
+    preservation: [
+      {
+        title: 'hi',
+        badges: ['asdjlakfsjlksaj', 'sadsadasd'],
+      },
+    ],
+  },
+    ],
+  };
+
+  render() {
+    const { name, country, city, website, images } = this.props;
+    const { reviews } = this.state;
+    return (
+      <>
+        <img
+          src={backgroundImage}
+          alt="background"
+          className="singlePlace-header-img"
+        />
+        <div className="buttons-container">
+          <Link to= "/search"><span className="buttons-container__back">
+            <Icon type="arrow-left" />
+            <span className="buttons-container__back__text">
+              Back to results
+            </span>
+          </span></Link>
+          <Link to="/review"><Button onClick={() => {}} className="buttons-container__add">
+            <Icon type="plus" style={{ color: '#fff' }} />
+            <span className="buttons-container__add__text">
+              Add your recommendation
+            </span>
+          </Button></Link>
+        </div>
+        <p className="heading-name-location">{`${name}, ${country}, ${city}`} </p>
+        <a href={website} className="heading-website">{website}</a>
+        <div className="singlePlace-slider">
+        <ImageCarousel slides={images} smallTitle />
+        </div>
+        <span className="recommended-by-text">Recommended by {reviews.length} contributors</span>
+        <Link to="/review"><Fab /></Link>
+        <div className="reviews-container">
+        {reviews.map(({id, reviewee, reviewDate, title, magicalFactors, tripDetails, reviewText, preservation})=>
+        <Review key={id} reviewee={reviewee} reviewDate={reviewDate} title={title} magicalFactors={magicalFactors} tripDetails={tripDetails} reviewText={reviewText} preservation={preservation} />
```

Comment:
this isnt how i wrote props to the review,
from the https://github.com/GSG-G7/sitspot/blob/6c2c3b9fcaf928d0f390966ccb161136420b798f/client/src/components/Review/index.js#L172-L174
you should just give it whole review object, without destructuring it